### PR TITLE
Add new WeekType Documentation

### LINKED ISSUE #12583: Add new Form WeekType
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#32061
| PR author(s) | @dFayet
| Merged in    | 4.4

### PR DESCRIPTION
**Pull request by dFayet**
Documents new https://github.com/symfony/symfony/pull/32061 feature
Fix #12583

### CODE REVIEW

File: reference/forms/types/week.rst [hunk 1]
```diff
@@ -0,0 +1,187 @@
+.. index::
+   single: Forms; Fields; WeekType
+
+WeekType Field
+==================
```
Comment (OskarStark) [inline reference/forms/types/week.rst]: Please shorten this underline to the length of the headline

File: reference/forms/types/week.rst [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+.. versionadded:: 4.4
+
+    The ``WeekType`` Type was introduced in Symfony 4.4.
```
Comment (OskarStark) [inline reference/forms/types/week.rst]: Type -> type

File: reference/forms/types/week.rst [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+This field type allows the user to modify data that represents a specific
+ISO 8601 week number  (e.g. ``1984-W05``).
```
Comment (OskarStark) [inline reference/forms/types/week.rst]: Please make ISO 8601 a link to:
https://en.m.wikipedia.org/wiki/ISO_8601

Please remove one whitespace before „(„

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by wouterj** [comment]
Thanks a lot for taking care of documenting your new feature! This is great documentation. I've now merged it and only did a few very very minor tweaks in 76076cfde.

Hope to see you back in the future with more contributions!

**Pull request merged**